docs/basics/primitives.md

```diff
@@ -39,7 +204,7 @@ that helps educate Phantom and simultaneously Cassandra of what your type `means
 "Deriving" is very simple, if we have a primitive for a type `T` that already exists, then we can define a new primitive
```

Comment:
remove paragraph for:
"Deriving" is very simple, if we have a primitive for a type `T` that already exists, then we can define a new primitive
for any type `X` we want, provided there is a bijection from `T` to `X`.